### DISCUSSION TITLE
Getting an error when I hit make -j2 which says I don't have certain packages but when I try to install those, the can't be found.

### DISCUSSION THREAD

**Discussion by GraphicsMonster** [idaholab/moose#25480] [Q&A Getting Started]
Here's everything that happened in the terminal. I am using ubuntu wsl btw.
`devansh@HP_15s:~/moose/framework$ make -j2
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/libtool: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/libtool: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
make: python-config: No such file or directory
make: python-config: No such file or directory
Traceback (most recent call last):
  File "/home/devansh/moose/framework/scripts/get_repo_revision.py", line 19, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
Checking if header needs updating: /home/devansh/moose/framework/include/base/MooseRevision.h...
Symlinking MOOSE configure /home/devansh/moose/framework/build/header_symlinks/MooseConfig.h

Failed to generate MooseRevision.h

/bin/sh: 1: /home/devansh/moose/libmesh/installed/contrib/bin/libmesh-config: not found
make: *** [/home/devansh/moose/framework/moose.mk:347: /home/devansh/moose/framework/include/base/MooseRevision.h] Error 1
make: *** Waiting for unfinished jobs....
devansh@HP_15s:~/moose/framework$ sudo apt install libmesh
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package libmesh
devansh@HP_15s:~/moose/framework$ sudo apt install libmes-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package libmes-dev
devansh@HP_15s:~/moose/framework$`

Any help will be much appreciated!

**Comment by GiudGiud** [comment]
Hello

we usually get this error when we forget to active the conda environment before building the code

Guillaume

**Reply by GraphicsMonster** [reply]
How do I do that? I've just cloned the moose repo and can see a conda directory in it. Do I need to set up conda again for wsl because I already use it natively.
Can you please break further steps down? Thanks a lot.

**Reply by GiudGiud** [reply]
mamba activate moose

you should have mostly followed the Linux instructions at this point, which make you install a conda/mamba environment named moose and that holds the moose dependencies
